README.md

```diff
@@ -216,3 +227,115 @@ Gets tax account information for a given nino and tax year
     }
 ```
 
+#### Get All Details
+Gets all information for a given nino and tax year. This includes:
+- all employments
+- for each employment that has any, a list of company benefits
+- for each employment that has any, a list of pay and tax details. Each pay and tax detail will also have a list of their Earlier Year Updates (EYUs).
+- for each employment that has any, a list of income sources
+- all allowances
+- the tax account information if available
+- state pension details if available
+
+| *URL* | *Supported Methods* | *Description* |
+|--------|----|----|
+| ```/:nino/:year/all-details``` | GET | Retrieves all details for a given nino and tax year. |
+
+**Parameters**
+
+|*Parameter*|*Required*|*Description*|
+|----|----|----|
+| ```:nino```| true | Standard National Insurance Number format e.g. QQ123456A |
+| ```:taxyear```| true | The first year in a tax year. e.g. for tax year 6th April 2016 - 5th April 2017 would be ```2016``` |
+
+**Return Format**
+```
+{
+  "employments": [
+    Employment{
+      "employmentId" : String UUID
+      "startDate" : LocalDate,
+      "endDate" : Option[LocalDate],
+      "payeReference" : String,
+      "employerName" : String,
+      "receivingOccupationalPension" : Boolean,
+      "receivingJobSeekersAllowance" : Boolean,
+      "employmentStatus" : Int (Live=1, PotentiallyCeased=2, Ceased=3, Unknown=99),
+      "worksNumber" : String,
+      "companyBenefitsURI" : Option[String],
+      "payAndTaxURI" : Option[String],
+      "employmentURI" : Option[String]
+    }
+  ],
+
+  "allowances" : [
+    Allowance{
+      "allowanceId": String UUID,
+      "iabdType": String,
+      "amount": BigDecimal
+    }
+  ],
+
+  "benefits" : {
```

Comment:
looks like this should sit in Employment, same for payAndTax and incomeSources, as they seem related and have a dependency on an employment and seems to be relevant inside of the Employment hierarchy.

Reply:
I suggested that exact format originally, but they've settled on the current format.